server/events/vcs/gitlab_client.go

```diff
@@ -531,8 +557,10 @@ func (g *GitlabClient) GetFileContent(pull models.PullRequest, fileName string)
 	opt := gitlab.GetRawFileOptions{Ref: gitlab.Ptr(pull.HeadBranch)}
 
 	bytes, resp, err := g.Client.RepositoryFiles.GetRawFile(pull.BaseRepo.FullName, fileName, &opt)
-	g.logger.Debug("GET /projects/%s/repository/files/%s/raw returned: %d", pull.BaseRepo.FullName, fileName, resp.StatusCode)
-	if resp.StatusCode == http.StatusNotFound {
+	if resp != nil {
+		g.logger.Debug("GET /projects/%s/repository/files/%s/raw returned: %d", pull.BaseRepo.FullName, fileName, resp.StatusCode)
+	}
+	if resp != nil && resp.StatusCode == http.StatusNotFound {
```

Comment:
I noticed this line as well was susceptible to a nil pointer deref, so I updated this one too.